charts/backstage/README.md.gotmpl

```diff
@@ -37,7 +37,7 @@ This chart bootstraps a [Backstage](https://backstage.io/docs/deployment/docker)
 
 ## Prerequisites
 
-- Kubernetes 1.25+ (OpenShift 4.12+)
+- Kubernetes 1.27+ ([OpenShift 4.14+](https://docs.redhat.com/en/documentation/openshift_container_platform/4.14/html/release_notes/ocp-4-14-release-notes#ocp-4-14-aws-security-groups))
```

Comment:
```suggestion
- Kubernetes 1.27+ ([OpenShift 4.14+](https://docs.redhat.com/en/documentation/openshift_container_platform/4.14/html/release_notes/index))
```

Reply:
you prefer the multi-page landing page to the all-in-one page that points specifically to the version of k8s on which OCP 4.14 is based? https://docs.redhat.com/en/documentation/openshift_container_platform/4.14/html-single/release_notes/index#ocp-4-14-aws-security-groups